### DISCUSSION TITLE
Is keycloak v22 still supported?

### DISCUSSION THREAD

**Discussion by mamccorm** [keycloak/keycloak#29126] [Q&A] [answered]
I don't see v22 listed in the official docs, nor is there release notes published, but it looks like there are periodic v22 tags being cut and incremented:
- https://github.com/keycloak/keycloak/tags

Could someone help clarify whether v22 is an officially maintained version stream? if not i'm curious why we are cutting tags for patch releases but this isn't reflected in the docs? 

Much appreciated

**Comment by codespearhead** [comment, marked as answer]
Keycloak 22 [1][2] is the first version of Keycloak with a corresponding commercial offering, making it effectively the project's first LTS release.

For the community edition, only the most recent stable release will receive updates [3][4].

Therefore, unless you're a paying customer, you should upgrade to the latest stable version as soon as it becomes available.

[1] https://access.redhat.com/announcements/7045552
[2] https://github.com/keycloak/keycloak/discussions/30548#discussioncomment-9826834
[3] https://github.com/keycloak/keycloak/discussions/10449#discussioncomment-2249604
[4] https://github.com/keycloak/keycloak/discussions/19937#discussioncomment-5720152

**Comment by mamccorm** [comment]
Interesting - thanks for the info, this was helpful
